RealmSwift/Combine.swift

```diff
@@ -685,37 +852,46 @@ public enum Publishers {
     /// should always be the first operation in the pipeline.
     ///
     /// Create this publisher using the `objectChangeset()` function.
-    public struct ObjectChangeset<T: Object>: Publisher {
+    public struct ObjectChangeset<O: Object>: Publisher {
```

Comment:
Probably could've kept this `T`.

Reply:
It can but below it is `ObjectChangesetWithToken<O: Object, T>`. Maybe for consistency it should be `O`?

Reply:
Maybe– I'm not sure there is mandated naming convention here. Generally speaking, `T` for type, `U` for the next type (and so on).  `K`/`Key` for key and `V`/`Value`. But if `O` gives you consistency, then I wouldn't be terribly pedantic. As you were.